### PR TITLE
add POSTGRES_SECRETS_TEMPLATE in container ENV

### PR DESCRIPTION
**Pull request by HenryKamg**
fixes: #717

### CODE REVIEW

File: charts/harbor-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -21,6 +21,8 @@ spec:
         command:
         - /manager
         env:
+        - name: POSTGRES_SECRETS_TEMPLATE
```
Comment (bitsf) [inline charts/harbor-operator/templates/deployment.yaml:24]: check if  it this can upgrade from v1.0.1

File: charts/harbor-operator/values.yaml [hunk 2]
```diff
@@ -23,6 +23,10 @@ imagePullSecrets: []
 # priorityClassName -- priority class to be used for the harbor-operator pods
 priorityClassName: ""
 
+postgresOperator:
+  env:
+    secrets: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
+
```
Comment (cndoit18) [inline charts/harbor-operator/values.yaml:29]:
```suggestion
postgres-operator:
  configKubernetes:
    secret_name_template: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
```
https://github.com/zalando/postgres-operator/blob/66620d5049d807656ad3322f87e88f9a33d56859/charts/postgres-operator/values.yaml#L157

File: config/helm/deployment/patch_deployment_config.yaml [hunk 3]
```diff
@@ -8,6 +8,8 @@ spec:
       containers:
       - name: manager
         env:
+        - name: POSTGRES_SECRETS_TEMPLATE
+          value: '{{ .Values.postgresOperator.env.secrets }}'
```
Comment (cndoit18) [inline config/helm/deployment/patch_deployment_config.yaml:12]:
```suggestion
          value: '{{ .Values.postgres-operator.configKubernetes.secret_name_template }}'
```

File: charts/harbor-operator/templates/deployment.yaml [hunk 4]
```diff
@@ -21,6 +21,9 @@ spec:
...
+          value: '{{ .Values.postgres-operator.configKubernetes.secret_name_template
+            }}'
```
Comment (cndoit18) [inline charts/harbor-operator/templates/deployment.yaml:26]:
```suggestion
          value: '{{ .Values.postgres-operator.configKubernetes.secret_name_template }}'
```

File: charts/harbor-operator/values.yaml [hunk 5]
```diff
@@ -23,6 +23,10 @@ imagePullSecrets: []
 # priorityClassName -- priority class to be used for the harbor-operator pods
 priorityClassName: ""
 
+postgres-operator:
+  configKubernetes:
+    secret_name_template: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
```
Comment (cndoit18) [inline charts/harbor-operator/values.yaml:28]:
```suggestion
    secret_name_template: "{username}.{cluster}.credentials"
```

**Review by cndoit18 — APPROVED** [review]
LGTM

**Review by steven-zou — APPROVED** [review]
LGTM

**Pull request merged**